pkgs/development/interpreters/ruby/default.nix

```diff
@@ -49,12 +49,12 @@ let
           rev    = tag;
           sha256 = sha256.git;
         } else fetchurl {
-          url = "http://cache.ruby-lang.org/pub/ruby/${ver.majMin}/ruby-${ver}.tar.gz";
```

Comment:
Method was introduced here: https://github.com/NixOS/nix/pull/721
This does not explain, why it happened to you in `17.03`.